composer.json

```diff
@@ -14,7 +14,7 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "phpunit/phpunit": "^4.0 || ^5.0 || ^6.0",
+        "phpunit/phpunit": "^5.7 || ^6.0",
```

Comment:
We need to keep 4.0 support for BC purposes at this point. That version still works with all versions of PHP; however, there's syntax from v4 that doesn't work later, which would mean that upping the PHPUnit version could break existing tests for people using this component.

I'll revert this on merge.